### PR TITLE
refactor(runtime): move performance API to timers extension

### PR DESCRIPTION
**Pull request by satyarohith**
cc @lucacasonato

### CODE REVIEW

File: extensions/timers/02_performance_util.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```
Comment (lucacasonato) [inline extensions/timers/02_performance_util.js:1]: This code is all duplicated with `webidl` extension / `Deno.core.serialize` / `Deno.core.deserialize`. The whole file could be removed with minor changes to performance.js.

**Comment by bartlomieju** [comment]
@lucacasonato ready to merge?

**Comment by lucacasonato** [comment]
No, need to still do some minor changes. Give me couple more hours :-)

**Review by lucacasonato — APPROVED** [review]
LGTM

**Pull request merged**